src/index.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ** This file is automatically generated by gapic-generator-typescript. **
+// ** https://github.com/googleapis/gapic-generator-typescript **
+// ** All changes to this file may be overwritten. **
+
+const gapic = Object.freeze({
+  v1beta1: require('./v1beta1'),
+  v1: require('./v1'),
+});
+module.exports = gapic.v1;
```

Comment:
I'm shocked that this provides proper types.  Has someone tried to `npm pack` this, and verify it works with TypeScript?

Reply:
We check it in our special [test application](https://github.com/googleapis/gapic-generator-typescript/blob/master/typescript/test/test_application_ts/package.json), but it's a great idea to have `npm pack` test here. We'll do it!

Reply:
To be clear - I don't think you need a test.  We do that in other places, and it's kinda clunky.  I just want to validate _once_ here.

Reply:
Hm, I actually thought having a pack test like the one in pub/sub is actually a good idea :)  I'll try to see if we can make it easy enough so that it does not randomly break things.

Reply:
Fixed that - now generating a pack test, I think it makes sense to check that the library can actually be used